source/fe/mapping_fe_field.cc

```diff
@@ -784,13 +779,13 @@ transform_unit_to_real_cell (const typename Triangulation<dim,spacedim>::cell_it
 //  Use the get_data function to create an InternalData with data vectors of
 //  the right size and transformation shape values already computed at point
 //  p.
-  update_internal_dofs(cell);
-
   const Quadrature<dim> point_quadrature(p);
   std_cxx11::unique_ptr<InternalData>
   mdata (dynamic_cast<InternalData *> (
            get_data(update_transformation_values, point_quadrature)));
 
```

Comment:
Even though not part of the patch: You can get rid of the dynamic_cast here by changing the return value of get_data to MappingFEField::InternalData, see #1183.